CONTRIBUTING.md

```diff
@@ -14,6 +14,12 @@ brew update
 ````
 brew install git
 ````
+
+## Install Python
+````
+brew install python
```

Comment:
Thanks for doing this! Can we change this to `brew install python3`?

Reply:
Yep! Updated.